ios/FluentUI/Core/Theme/Tokens/ControlTokenSet.swift

```diff
@@ -134,6 +150,19 @@ public enum ControlTokenValue {
                                        disabled: defaultColor)
         }
     }
+
+    public var pillButtonDynamicColors: PillButtonDynamicColors {
+        if case .pillButtonDynamicColors(let pillButtonDynamicColors) = self {
+            return pillButtonDynamicColors()
+        } else {
+            assertionFailure("Cannot convert token to PillButtonDynamicColors: \(self)")
+            let defaultColor = DynamicColor(light: ColorValue(0xE3008C))
```

Comment:
Ah good point. It comes from the global `Hot Pink` primary color, which should hopefully stand out. That said, I think I want to fail a little more quietly here, so I'll put pink behind a debug flag and use black otherwise.